build.gradle

```diff
@@ -273,6 +273,7 @@ task python37PostCommit() {
 
 task python38PostCommit() {
   dependsOn ":sdks:python:test-suites:portable:py38:crossLanguagePythonJavaKafkaIOFlink"
+  dependsOn ":sdks:python:test-suites:portable:py38:crossLanguagePythonJavaJdbcIO"
```

Comment:
Ah sorry, I see now that this is how you know it's running on Flink and Spark :)